### PR TITLE
Gateway Editor Fix

### PR DESCRIPTION
**Pull request by drivera73**
The gateway editor makes an erroneous assumption which results in a code error when the interval isn't set (i.e. allowed to default), but the time period is.

This fix addresses that.

Judging by how the rest of the code is written, it might be a good idea to do give it a once-over so that there's proper input defense is applied.

### CODE REVIEW

File: src/etc/inc/plugins.inc.d/openvpn/ovpn-linkup [hunk 1]
```diff
@@ -3,27 +3,33 @@
...
+/usr/bin/touch "/tmp/${1}upv6"
 
-/usr/local/opnsense/service/configd_ctl.py interface newip $1
+/usr/local/opnsense/service/configd_ctl.py interface newip "${1}"
+
+# If there's a setting for "additional --up script", then run that, but
+# within a subshell so we don't mess up the containing environment,
+# and we don't cause problems for the other stuff
 
 exit 0
```
Comment (drivera73) [inline src/etc/inc/plugins.inc.d/openvpn/ovpn-linkup]: This is a stowaway due to poor branch management on my part... it's still useful, but not essential to the spirit of this change request. Cherry pick at your discretion. :)

**Comment by AdSchellevis** [comment]
@drivera73 can you try https://github.com/opnsense/core/commit/0ec7e1f804537c39931848c1522a1c5902df3570 ?

**Comment by drivera73** [comment]
Patch 0ec7e1f fixes the issue! When will it be released? 20.1.3? 20.7?

**Comment by AdSchellevis** [comment]
@drivera73 thanks for confirming, probably in a 20.1.x version, maybe 20.1.5

**Pull request closed without merging**